### PR TITLE
Add return list of individual counts to find_node_in_area

### LINKED ISSUE #2326: find_nodes_in_area expansion
I like using the find_nodes_in_area command for certain routines which returns either a list of positions or # of nodes found, but sometimes I wish it would return how many of EACH node it has found ..or.. a list of nodenames to go with the positions...

Feature 1:
```
nodes, dirt, stone = minetest.find_nodes_in_area(minp, maxp, {"default:dirt", "default:stone"})
print ("dirt = ", dirt, "stone = ", stone)
```
Feature 2:
```
nodes = minetest.find_nodes_in_area(minp, maxp, {"default:dirt", "default:stone"})
local dirt, stone = 0
for _, nod in ipairs(nodes)
   if nodes[nod].nodename == "default:dirt" then dirt = dirt + 1 end
   if nodes[nod].nodename == "default:stone" then stone = stone + 1 end
end
print ("dirt = ", dirt, "stone = ", stone)
```

### PR DESCRIPTION
**Pull request by TeTpaAka**
Fixes #2326

### CODE REVIEW

File: src/script/lua_api/l_env.cpp [hunk 1]
```diff
@@ -572,19 +572,28 @@ int ModApiEnvMod::l_find_nodes_in_area(lua_State *L)
...
-		if(filter.count(c) != 0) {
-			push_v3s16(L, p);
-			lua_rawseti(L, -2, ++i);
-		}
+	for (s16 x = minp.X; x <= maxp.X; x++)
+		for (s16 y = minp.Y; y <= maxp.Y; y++)
+			for (s16 z = minp.Z; z <= maxp.Z; z++) {
+				v3s16 p(x, y, z);
+				content_t c = env->getMap().getNodeNoEx(p).getContent();
+				if(filter.count(c) != 0) {
```
Comment (est31) [inline src/script/lua_api/l_env.cpp]: space :)

**Comment by est31** [comment]
Nice addition!

File: src/script/lua_api/l_env.cpp (hunk 1 above)
Comment (TeTpaAka) [inline src/script/lua_api/l_env.cpp]: Fixed. I should really pay more attention while fixing the style.

Comment (est31) [inline src/script/lua_api/l_env.cpp]: Everybody makes mistakes, its no catastrophy :)

**Comment by est31** [comment]
Code looks good :+1:

**Comment by est31** [comment]
e50aa4ed06f36c74a892ec68d576c52ba9dc0b2c

**Pull request closed without merging**